src/components/Form.js

```diff
@@ -61,8 +61,7 @@ export default class Form extends Component {
 
   onChange = (formData, options={validate: false}) => {
     const liveValidate = this.props.liveValidate || options.validate;
-    const errors = liveValidate ? this.validate(formData) :
-                                  this.state.errors;
+    const errors = liveValidate ? this.validate(formData) : [];
```

Comment:
I'll add a checkbox in the playground to enable/disable live validation so we'll figure out how usable it this.